interface/usergroup/mfa_registrations.php

```diff
@@ -0,0 +1,119 @@
+<?php
+/**
+ * Multi-Factor Authentication Management
+ *
+ * @package   OpenEMR
+ * @link      http://www.open-emr.org
+ * @author    Rod Roark <rod@sunsetsystems.com>
+ * @copyright Copyright (c) 2018 Rod Roark <rod@sunsetsystems.com>
+ * @license   https://github.com/openemr/openemr/blob/master/LICENSE CNU General Public License 3
+ */
+
+require_once('../globals.php');
+
+use OpenEMR\Core\Header;
+
+function writeRow($method, $name)
+{
+    echo "        <tr><td>&nbsp;";
+    echo text($method);
+    echo "&nbsp;</td><td>&nbsp;";
+    echo text($name);
+    echo "&nbsp;</td><td>";
+    echo "<input type='button' onclick='delclick(\"" . attr($method) . "\", \"" .
+        attr($name) . "\")' value='" . xla('Delete') . "' />";
```

Comment:
In these weird cases where  nesting php variable within a javascript literal that is nested within html code, can use:
```
echo "<input type='button' onclick='delclick(\"" . attr(addslashes($method)) . "\", \"" .
        attr(addslashes($name)) . "\")' value='" . xla('Delete') . "' />";
```